src/profile-logic/tracks.js

```diff
@@ -115,12 +115,19 @@ function _getDefaultLocalTrackOrder(tracks: LocalTrack[], profile: ?Profile) {
 
     // If the tracks are both threads, sort them by thread name, and then by
     // creation time if they have the same name.
-    if (tracks[a].type === 'thread' && tracks[b].type === 'thread' && profile) {
+    if (
+      (tracks[a].type === 'thread' || tracks[a].type === 'marker') &&
+      (tracks[b].type === 'thread' || tracks[b].type === 'marker') &&
+      profile
+    ) {
```

Comment:
I feel like this change shouldn't be part of this pull request (?).

Reply:
It's related to the disable reordering.

Reply:
You're right, it is related to my PR #4198